itou/approvals/migrations/0013_prolongation_create_trigger.py

```diff
@@ -0,0 +1,69 @@
+# Generated by Django 3.1.3 on 2020-12-05 16:59
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("approvals", "0012_auto_20210203_1456")]
+
+    operations = [
+        migrations.RunSQL(
+            sql="""
+                CREATE OR REPLACE FUNCTION update_approval_end_at_for_prolongation()
```

Comment:
Du coup il faut peut-être renommer l'ancienne fonction de `update_approval_end_at` à `update_approval_end_at_for_suspension` non?

Reply:
Peut-être par soucis d'homogénéité… Dès que ma flemme passe je regarde si c'est pas trop chiant de renommer un _trigger_ déjà en place :)